src/domain/event/EventUtils.ts

```diff
@@ -403,3 +409,10 @@ export const getAudienceAgeText = (
   }`;
   return `${ageLimit} -${t('event.info.age')}`;
 };
+
+export const getEventTypeByEventTypeId = (
+  eventTypeId: EventTypeId
+): EventType =>
+  Object.keys(EVENT_TYPE_TO_ID).find(
+    (eventType) => EVENT_TYPE_TO_ID[eventType as EventType] === eventTypeId
+  ) as EventType;
```

Comment:
I tried to avoid that the change is needed in multiple places. I understand that it's hard to understand what it does, but the thing is that you should never touch it :) @Sartonon, I can change it to your suggestion if you really wish so.

Reply:
Added a function description.